pom.xml

```diff
@@ -32,6 +32,7 @@
     <module>testing/camunda-process-test-java</module>
     <module>testing/camunda-process-test-spring</module>
     <module>clients/java</module>
+    <module>qa</module>
```

Comment:
I forgot to tell you to remove this
this should be controlled with `skipQaBuild` property (see `integration-tests-build` below)